docs/user-guide/zonal-average.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>All bold. Probably only the first parts are to be bold instead. Also, looks like some text editing is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>